ZenPacks/zenoss/ZenPackLib/lib/base/ComponentBase.py

```diff
@@ -151,27 +150,27 @@ def setIdsInRelationship(self, relationship, ids):
             try:
                 component = result.getObject()
             except Exception as e:
-                LOG.error("Trying to access non-existent object %s", e)
+                self.LOG.error("Trying to access non-existent object %s", e)
             else:
                 obj_map[result.id] = component
 
         for id_ in new_ids.symmetric_difference(current_ids):
             obj = obj_map.get(id_)
             if not obj:
-                LOG.error(
+                self.LOG.error(
                     "setIdsInRelationship (%s): No targets found matching "
                     "id=%s", relationship, id_)
 
                 continue
 
             if id_ in new_ids:
-                LOG.debug("Adding %s to %s" % (obj, relationship))
+                self.LOG.debug("Adding %s to %s" % (obj, relationship))
```

Comment:
At some point we're going to have to be python 3 compatible, so use .format() instead of %.  

`"Adding {} to {}".format(obj, relationship)`